content/ecosystem/paas-devops-platforms/nullstone.md

```diff
@@ -0,0 +1,28 @@
++++
+title="Nullstone"
+aliases="/frameworks/nullstone/"
```

Comment:
Can you please remove this line? It's not needed in the header.

Reply:
Ah, I copied from the format of others. I will drop this line.